src/prost/wrapper_eraftpb.rs

```diff
@@ -0,0 +1,922 @@
+impl Entry {
+    pub fn new_() -> Entry {
+        ::std::default::Default::default()
+    }
+    pub fn clear_entry_type(&mut self) {
+        self.entry_type = 0
+    }
+    pub fn set_entry_type_(&mut self, v: EntryType) {
+        self.entry_type = unsafe { ::std::mem::transmute::<EntryType, i32>(v) };
+    }
+    pub fn get_entry_type(&self) -> EntryType {
+        unsafe { ::std::mem::transmute::<i32, EntryType>(self.entry_type) }
+    }
+    pub fn clear_term(&mut self) {
+        self.term = 0
+    }
+    pub fn set_term(&mut self, v: u64) {
+        self.term = v;
+    }
+    pub fn get_term(&self) -> u64 {
+        self.term
+    }
+    pub fn clear_index(&mut self) {
```

Comment:
Seems a lot of these would benefit from `#[inline]`?

Reply:
nrc/kvproto-build#3